lib/veda_idmatrix/request.rb

```diff
@@ -113,14 +111,19 @@ def id_matrix_operation
     details[:'current-address'] = current_address
     details[:'phone']=  phone
     details[:'email-address'] = email_address
-    details[:'drivers-licence-details'] = drivers_licence_details
-    details[:'passport-details'] = passport_details
-    # Have to exclude medicare if details missing #5519
-    details[:'medicare'] = medicare_details unless medicare_details.empty?
+
```

Comment:
you cannot simply decide that all 3 of these sections should now be excluded when blank. They should always be present, with the exception of medicare, there is even a comment in the code about why this is the case.